### DISCUSSION TITLE
Copilot said my subscription has ended but I have GitHub Pro

### DISCUSSION THREAD

**Discussion by tchu96** [community/community#40049] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Copilot was working fine and it suddenly stopped working for me. 

I have GitHub Pro so I should be able to use Copilot for free. However, it suddenly stopped working for a while back and told me that my subscription has ended (as the image below).
![image](https://user-images.githubusercontent.com/95496129/203924396-9c0c6956-860d-4188-879b-200e51013fb1.png)

When I press view->Output->GitHub Copilot in Visual Studio Code, it also says my subscription has ended.

In my billing tab on GitHub, it says that I am eligible to use Copilot for free (as the image below) which contradicts with the first image.
![image](https://user-images.githubusercontent.com/95496129/203924001-a34425b3-f2f6-494a-bd97-3b6214aaec90.png)

I cannot unsubscribe to co-pilot and re-subscribe in the billing tab because there is no such option. 

How could I fix this? I have searched online for similar problems and I can't seem to find any solution since their problems are either not answered, Copilot suddenly works again somehow, or are fixed by Copilot support.

I've tried:
- logging out and in on GitHub
- reinstalling Copilot
- rollback to previous Copilot versions
- logging out of GitHub on Visual Studio Code

**Comment by Akash1134** [comment]
Hello @fulyze We're sorry for the trouble here. The best place to go for answers to account-specific questions like this one is by opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them and they should be able to help you get this sorted :)

**Comment by Akash1134** [comment, marked as answer]
Just an update. You might be facing this issue because of a bad server deployment that affected some users. The issue has now been resolved. Please checkout this [discussion](https://github.com/orgs/community/discussions/40026).
